src/app/app.component.ts

```diff
@@ -1,30 +1,69 @@
-import { Component } from '@angular/core';
+import { Component } from "@angular/core";
 
-import { Platform } from '@ionic/angular';
-import { SplashScreen } from '@ionic-native/splash-screen/ngx';
-import { StatusBar } from '@ionic-native/status-bar/ngx';
-import { DataSharingService } from '../services/data-sharing.service';
-import { UserServices } from 'src/services/user.services';
-import { TranslationService } from '../services/translation.service';
+import { Platform, AlertController } from "@ionic/angular";
+import { SplashScreen } from "@ionic-native/splash-screen/ngx";
+import { StatusBar } from "@ionic-native/status-bar/ngx";
+import { DataSharingService } from "../services/data-sharing.service";
+import { UserServices } from "src/services/user.services";
+import { TranslationService } from "../services/translation.service";
+import { Router } from "@angular/router";
+import { GoogleMapComponent } from "./components/outdoor/google-map/google-map.component";
 
```

Comment:
Nice feature :D 

As angel commented, we are using a single quote for formatting in the entire project :O Check out the wiki for how to change it https://github.com/claudiiafg/MapConU/wiki/Formatting.

I also get the same bug mentionned by angel, but I'm on an emulator

![image](https://user-images.githubusercontent.com/39540423/77723566-36fdd000-6fc7-11ea-835a-48e78712f837.png)

The phone is a pixel 2. What is yours @achung98 and what's yours @bmaryam ??